src/main/java/ui/windows/StatsRendererData.java

```diff
@@ -0,0 +1,33 @@
+package ui.windows;
```

Comment:
StatsRendererData is the model in the model view controller pattern, and usually all parts of MVC are in the outermost layer since they're more low-level details.